cypress/integration/tests/checkout/checkout.js

```diff
@@ -0,0 +1,31 @@
+import Chance from 'Chance';
+import CheckoutPage from '../../../page-objects/checkoutPage';
+import products from '../../../fixtures/products.json';
+import ShopPage from '../../../page-objects/shopPage';
+
+const chance = new Chance();
+
+describe('Checkout', () => {
+  beforeEach(() => {
+    cy.wrap(ShopPage.pickRandomProduct(products)).as('randomProduct');
+  });
+
+  it('TA-20: Cart items count should be equal to number of added products', function () {
+    const quantity = chance.integer({ min: 1, max: 12 });
```

Comment:
please try to do the same with `lodash` which is bundled into Cypress

Reply:
I'm in love with it. So easy and readable and doesn't require additional imports. TY <3